docs/en/writing-running-appium/ios/ios-tvos.md

```diff
@@ -0,0 +1,69 @@
+## tvOS support
+
+Appium 1.13.0+ has tvOS support via XCUITest driver.
+
+## Setup
+
+You can run tests for tvOS changing `platformName` section in capability like below.
+
+```json
+{
+    "automationName": "XCUITest",
+    "platformName": "tvOS",
+    "platformVersion": "12.2",
+    "deviceName": "Apple TV",
+    ...
+}
+```
+
+## Limitations
+Gesture commands do not work for tvOS. Some commands such as pasteboard also do not work.
+
+We can handle `focus` on tvOS simply pressing keys such as up/down/left/right/home.
+tvOS does actions against the _focused_ element. You can get the focus attribute as [getting attribute API](http://appium.io/docs/en/commands/element/attributes/attribute/).
+
+
+## Basic Actions
+
+```ruby
+# Ruby
+
+element = @driver.find_element :accessibility_id, 'element on the app'
+# Returns if the element is focused or not
+element.focused #=> 'true'
+# Appium moves the focuse to the element with pressing keys automatically and clicks it.
```

Comment:
the focus to the element by pressing the corresponding keys and clicking the element